src/components/form/Form.ts

```diff
@@ -1,54 +1,74 @@
 import BaseBlock from '../../common/base-block';
 import template from './template.hbs';
-import { BaseBlockOptions, BaseBlockProps } from '../../common/types';
+import {BaseBlockOptions, BaseBlockProps} from '../../common/types';
 import Validator from '../../helpers/validator';
 
-type FormOptions = BaseBlockOptions & {
+export type FormOptions = BaseBlockOptions & {
     validators?: Record<string, Validator>
+    onSubmit?: (data: Record<string, any>) => any,
```

Comment:
может лучше использовать void?
```suggestion
    onSubmit?: (data: Record<string, any>) => void,
```